samples/bluetooth/rpc_host/README.rst

```diff
@@ -23,7 +23,8 @@ Overview
 ********
 
 .. note::
-   Currently, serialization of the :ref:`zephyr:bt_gap` and :ref:`zephyr:bluetooth_connection_mgmt` is supported.
+   Currently, the serialization of all Bluetooth APIs is not supported.
```

Comment:
This sentence makes me pause when I read it. I suggest softening "all" to "some" for easier reading

```suggestion
   Currently, the serialization of some Bluetooth APIs is not supported.
```

..or using a cliche phrasing, like this:

```suggestion
   Only a subset of Zephyr's Bluetooth APIs are available trough Bluetooth nRF RPC for now.
```

Reply:
Or maybe even something like this:
Currently, only a subset of Zephyr's Bluetooth APIs **is** available over Bluetooth nRF RPC.

Reply:
I rephrased it as you suggested. Please rereview